File: snippets/c/array/set/set.c
#include<stdio.h>

int main() {
    int array[] = {1,2,3,4,5};
    int index = 1;
    int item = 100;
    array[index] = item;
    for(int i=0;i<5;i++){
        printf("%d \n", array[i]);
    } 
   return 0;
}

File: snippets/java/array/set/Set.java
package array;

import java.util.Arrays;

public class Set {

    public static void main(String args[]) {
        int[] array = new int[]{5, 4, 3, 2, 1};
        int index = 1;
        int item = 100;
        array[index] = item;  // ≡
        System.out.println(Arrays.toString(array));
    }
}
